src/styles.css

```
@@ -142,6 +142,8 @@ body.modal-open{overflow:hidden;  height:100vh;}
 .fetch-btn:hover{background:var(--orange);color:white}
 .fetch-btn:disabled{opacity:.5;cursor:not-allowed}
 .spin{width:10px;height:10px;border:2px solid rgba(255,255,255,.3);border-top-color:#fff;border-radius:50%;animation:spin .7s linear infinite}
+.spin{width:10px;height:10px;border:2px solid rgba(255,255,255,.3);border-top-color:#fff;border-radius:50%;animation:spin .7s linear infinite}
```

Stylelint (17.11.1)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)